dev/preview/infrastructure/modules/gce/vm.tf

```diff
@@ -70,8 +81,9 @@ data "kubernetes_secret" "harvester-k3s-dockerhub-pull-account" {
 }
 
 locals {
+  vm_name = "preview-${var.preview_name}"
```

Comment:
prefixing the VM name with `preview-` such that the packer image sets up the preview LVM volumes, which conditionally runs based off the hostname (see https://github.com/gitpod-io/gitpod-packer-gcp-image/pull/240/files#diff-167633e1acfa7e4989f5ba35a6f0847b825b2da186c0a0dd9b1f5a12ff752202R4)